flutter/android/app/build.gradle

```diff
@@ -43,7 +43,7 @@ android {
 
     defaultConfig {
         applicationId "org.mlcommons.android.mlperfbench"
-        minSdkVersion 16
+        minSdkVersion 21
```

Comment:
I need to increase the `minSdkVersion` to 21 to fix a build error described here https://developer.android.com/build/multidex

Per this screenshot (from Android Studio) this should not affect much the number of Android phones which can run this app.

<img width="300" alt="Screenshot 2023-05-03 at 22 40 25" src="https://user-images.githubusercontent.com/85728587/235967039-e20de55a-cd02-410c-808a-46c38eae3207.png">